test/e2e/cypress/integration/profiles/code/login/error.spec.ts

```diff
@@ -160,6 +160,12 @@ context("Login error messages with code method", () => {
             "contain",
             "Property identifier is missing",
           )
+        } else if (app === "react") {
+          // The backspace trick is not working in React.
+          cy.get('[data-testid="ui/message/4010008"]').should(
```

Comment:
The React UI does not really allow deleting input fields, but we're also fine if this flow fails with another failure reason (invalid code).